doc/en/user/source/extensions/css/workshop/point.rst

```diff
@@ -0,0 +1,708 @@
+            "${if_then_else(equalTo(FEATURECLA,'Admin-0 capital'),'star','circle')}"
+          );
+       }
+     
+   Which is represented in SLD as:
+  
+   .. code-block:: xml
+  
+      <sld:PointSymbolizer>
+        <sld:Graphic>
+           <sld:Mark>
+              <sld:WellKnownName>${if_then_else(equalTo(FEATURECLA,'Admin-0 capital'),'star','circle')}</sld:WellKnownName>
+              <sld:Fill/>
+              <sld:Stroke/>
+           </sld:Mark>
+        </sld:Graphic>
+      </sld:PointSymbolizer>
+
+   An example is available here :download:`point_example.css </files/point_example2.css>`
+
+#. We can use a **Layer Group** to explore how symbology works together to form a map.
+  
+   * ne:NE1
+   * ne:states_provincces_shp
+   * ne: populated_places
+
+#. To help start things out here is a style for ``ne:states_provinces_shp``:
+
+   .. code-block:: css
+
+       * {     
+          fill: white,[
+           recode(mapcolor9,
+             1,'#8dd3c7', 2,'#ffffb3', 3,'#bebada',
+             4,'#fb8072', 5,'#80b1d3', 6,'#fdb462',
+             7,'#b3de69', 8,'#fccde5', 9,'#d9d9d9')
+          ];
+          fill-opacity: 05%,50%;
+       
+          stroke: black;
+          stroke-width: 0.25;
+          stroke-opacity: 50%;
+       }
+
+#. This background is relatively busy and care must be taken to ensure both symbols and labels are clearly visible.
+
+   Here is an example with labels:
+
+   .. image:: img/point_challenge_1.png
+
+   We can reuse the halo technique here to make things readable. Here is an example of the css:
+
+   .. code-block:: css
+
+       * {
+          mark-size: [5+((10-SCALERANK)/3)];
+
+          font-fill: black;
+          font-family: "Arial";
+          font-size: 10;
+
+          label-anchor: 0.5 1;
+          label-offset: 0 [-12+SCALERANK];
+
+          halo-radius: 2;
+          halo-color: lightgray;
+          halo-opacity:0.7;
+
+          -gt-mark-label-obstacle: true;
+          -gt-label-max-displacement: 90;
+          -gt-label-priority: [0 - LABELRANK];
+       }
+       :symbol {
+         fill: black;
+         stroke: white;
+         stroke-opacity:0.75;
+       }
+
+#. In addition to image formats GeoServer can make use other kinds of graphics, such as True Type fonts:
+
+   .. code-block:: css
+
+       * {
+          mark: symbol("ttf://Webdings#0x0064");
+       }
+       :mark {
+          stroke: blue;
+       }
+     
+#. Additional fonts dropped in the :file:`styles` directory are available for use.
+
+#. The GeoServer rendering engine allows Java developers to hook in additional symbol support.
+  
+   This facility is used by GeoServer to offer the shapes used for pattern fills. Community extensions allow the use of simple custom shapes and even charts.
+
+#. In GeoServer 2.6 support has been added for custom grpahics using the WKT Geometry representation. If you would like to try this functionality in earlier versions of GeoServer look up for the GeoTools WKT plugin.
```

Comment:
Please fix this sentence for legibility and spelling. Since 2.6 is current, not sure I'd even worry about the "in earlier versions" part. So maybe just replace with: "You can also create custom graphics using Well-Known Text (WKT) representation"